test/utils/utils.spec.ts

```diff
@@ -0,0 +1,49 @@
+/* tslint:disable: only-arrow-functions ter-prefer-arrow-callback */
+
+import * as chai from "chai";
+import { parseStrToNumStrict } from "../../app/utils/utils";
+
+describe("utils/utils.ts - parseStrToNumStrict", () => {
+  it("should return Nan for null", function() {
+    const parsed = parseStrToNumStrict(null);
+    chai.assert.isNaN(parsed);
```

Comment:
I would really prefer to stick to `expect(something).to.be.something` API. We had the same discussion on smart contract project.